provisioning/ci/resources/aws.yaml

```diff
@@ -0,0 +1,87 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Terraform user & remote state backend
+Parameters:
+  BackendPrefix:
+    Type: String
+    Description: Prefix for all Terraform Backend resources
+
+Resources:
+  TerraformRemoteStateS3Bucket:
```

Comment:
Why do we need S3?

Reply:
No clue, maybe we don't. It was copied from [keboola/object-encryptor@main/provisioning/ci/resources/aws.yaml](https://github.com/keboola/object-encryptor/blob/main/provisioning/ci/resources/aws.yaml)